### ISSUE TITLE
Skip loading parameter wh.2.bias when predicting with fairmot_dla34.pth

### ISSUE THREAD

**Issue by shengjie-shi** [ifzhang/FairMOT#239]
When I was doing evaluation on MOT16 test set, it gave me the following message after loading the model:
Skip loading parameter wh.2.weight and so on...
It says "your model does not fully load the pre-trained weight".

But it does not happen when I do eval with the former offered all_dla34.pth model. 

So could you please tell me why this problem occurs and how to fix it? Thanks a lot!

**Comment by ifzhang** [comment]
You can use the new code now. The wh head of the new model has 4 channels (left, top, right, bottom).

**Comment by shengjie-shi** [comment]
thank u very much! I would try it out
